package/MDAnalysis/transformations/fit.py

```diff
@@ -0,0 +1,241 @@
+    AtomGroup in a trajectory. A plane can be given using the flag `plane`
+    so that only translations and rotations in that particular plane are
+    removed. This is useful for protein-membrane systems to where the membrane
+    must remain in the same orientation.
+    
+    Example
+    -------
+    Removing the translations and rotations of a given AtomGroup `ag` on the XY plane
+    by fitting it to a reference `ref`, using the masses as weights for the RMSD fit.
+    
+    ..code-block::python
+    
+        ag = u.select_atoms("protein")
+        ref = mda.Universe("reference.pdb")
+        transform = MDAnalysis.transformations.fit_rot_trans(ag, ref, plane="xy", weights="mass")
+        u.trajectory.add_transformations(transform)
+    
+    Parameter
+    ---------
+    ag : Universe or AtomGroup
+       structure to translate, a
+       :class:`~MDAnalysis.core.groups.AtomGroup` or a whole 
+       :class:`~MDAnalysis.core.universe.Universe`
+    reference : Universe or AtomGroup
+       reference structure, a :class:`~MDAnalysis.core.groups.AtomGroup` or a whole 
+       :class:`~MDAnalysis.core.universe.Universe`
+    plane: str, optional
+        used to define the plane on which the translations will be removed. Defined as a 
+        string of the plane. Suported planes are "yz", "xz" and "xy" planes.
+    weights : {"mass", ``None``} or array_like, optional
+       choose weights. With ``"mass"`` uses masses as weights; with ``None``
+       weigh each atom equally. If a float array of the same length as
+       `ag` is provided, use each element of the `array_like` as a
+       weight for the corresponding atom in `mobile`.
+       
+    Returns
+    -------
+    MDAnalysis.coordinates.base.Timestep
+    """
+    if plane is not None:
+        if plane not in ('xy', 'yz', 'xz'):
+            raise ValueError('{} is not a valid plane'.format(plane))
+        axes = {'yz' : 0, 'xz' : 1, 'xy' : 2}
+        plane = axes[plane]
+    try:
+        if ag.atoms.n_residues != reference.atoms.n_residues:
+            raise ValueError("{} and {} have mismatched number of residues".format(ag,reference))
+    except AttributeError:
+        raise AttributeError("{} or {} is not valid Universe/AtomGroup".format(ag,reference))
+    ref, mobile = align.get_matching_atoms(reference.atoms, ag.atoms)
+    try:
+        weights = align.get_weights(ref.atoms, weights=weights)
+    except (ValueError, TypeError):
+        raise TypeError("weights must be {'mass', None} or an iterable of the "
+                        "same size as the atomgroup.")
+    ref_com = ref.center(weights)
+    ref_coordinates = ref.atoms.positions - ref_com   
+    def wrapped(ts):
+        mobile_com = mobile.atoms.center(weights)
+        mobile_coordinates = mobile.atoms.positions - mobile_com
+        rotation, dump = align.rotation_matrix(mobile_coordinates, ref_coordinates, weights=weights)
+        if plane:
```

Comment:
If your plane in `yz`, then `plane` will be 0 which is the same as `False`.

Reply:
It doesn't look this crude in the source code anymore--`if plane is not None` is used now, along with some other code--so I'll mark this specific point as resolved.